calcom/cal.com
# fix: Not able to add a second Outlook Calendar

## What does this PR do?

Allow user to add second or more outlook account, fixing the error of duplicate adding of the existing account

Fixes #11830

Loom Video for fix
https://www.loom.com/share/57b2f1b4a3ba47f2a8f446c84bc20aba?sid=3aaf99f3-5feb-48e3-8e03-1dbd9c576209

## Requirement/Documentation

## Type of change




## How should this be tested?

Add additional outlook accounts and make sure the new outlook accounts are added.

## Mandatory Tasks


## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
